src/elements/OcSidebar.vue

```diff
@@ -84,7 +87,7 @@ export default {
      */
     navItems: {
       type: Array,
-      required: true
+      required: false
```

Comment:
Please add an empty array as default. Otherwise there would be an error when `navItem` is not provided and the `maintContent` slot is not used.
like this: `default: () => []`

Reply:
Done